hazelcast-jet-all/pom.xml

```diff
@@ -127,10 +90,80 @@
                             <artifactId>org.osgi.core</artifactId>
                             <version>4.2.0</version>
                         </additionalDependency>
-
                     </additionalDependencies>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <version>${maven.bundle.plugin.version}</version>
+                <executions>
+                    <execution>
+                        <id>bundle-manifest</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>bundle</goal>
+                        </goals>
+                        <configuration>
+                            <supportedProjectTypes>jar</supportedProjectTypes>
+                            <instructions>
+                                <Embed-Dependency>hazelcast-jet-core;inline=true</Embed-Dependency>
+                                <Embed-Transitive>true</Embed-Transitive>
+                                <Implementation-Title>${project.name}</Implementation-Title>
+                                <Implementation-Version>${project.version}</Implementation-Version>
+                                <Implementation-Vendor-Id>${project.groupId}</Implementation-Vendor-Id>
+                                <Implementation-Vendor>${project.organization.name}</Implementation-Vendor>
+                                <Implementation-URL>${project.url}</Implementation-URL>
+                                <Specification-Title>${project.name}</Specification-Title>
```

Comment:
I believe these are not needed.

Reply:
I added because these fields were present in the previous Manifest, but I can remove them